jsconfig.eslint-base.json

```diff
@@ -9,5 +9,5 @@
     "moduleResolution": "node"
   },
   "include": [],
-  "exclude": ["packages/test262-runner/test262"]
+  "exclude": ["packages/test262-runner/test262", "**/*.d.ts"]
```

Comment:
Wouldn't that prevent linting of explicit / manual `.d.ts` files?

Reply:
Ah, yes.  I've narrowed it to `**/exports.d.ts`.

Reply:
Oh, no.  I don't know how to solve this.  `exports.d.ts` needs to be included in order for its manual types to take effect, but if it *is* included, then `exports.js` has the unsilenceable eslint error.

Where do we go from here?

Reply:
I'm confused because I think @warner did not experience this issue in https://github.com/Agoric/agoric-sdk/pull/8075

Reply:
> I'm confused because I think @warner did not experience this issue in [Agoric/agoric-sdk#8075](https://github.com/Agoric/agoric-sdk/pull/8075)

Swing-store does not include `*.ts` in its tsconfig files, so as you say:

> Wouldn't that prevent linting of explicit / manual `.d.ts` files?

Yes, yes it would.

```
snow:swing-store michael$ cat jsconfig.json 
// This file can contain .js-specific Typescript compiler config.
{
  "extends": "../../tsconfig.json",
  "include": [
    "src/**/*.js",
    "test/**/*.js",
  ],
}
snow:swing-store michael$ cat ../../tsconfig.json 
{
  "compilerOptions": {
    "target": "esnext",
    "module": "esnext",
    "moduleResolution": "node",
    "skipLibCheck": true,

    "allowJs": true,
    "checkJs": true,
    "strict": false,
    "downlevelIteration": true,
    "strictNullChecks": true,
    "noImplicitThis": true,

    "noEmit": true
  },
  "include": [
    ".eslintrc.cjs"
  ]
}
snow:swing-store michael$
```

Reply:
Until I've tested against the latest typescript-eslint and/or filed a bug against that repository, I've opted for limiting the damage just to `exports.d.ts`.

Reply:
I'm confused now, what did we settle on for the exclusion?

Reply:
> what did we settle on for the exclusion?

[Omitting `src/**/exports.js`](https://github.com/endojs/endo/pull/1690/files#diff-ff983a2f596fa583a661fbfceba5f89dc87af77d8a3ac76a42c3356972ee085eR52) from the typescript-eslint configuration.